PsimagLite/examples/CMakeLists.txt

```diff
@@ -4,3 +4,9 @@ add_executable(testLanczos testLanczos.cpp)
 target_link_libraries(testLanczos psimaglite)
 
 add_test(NAME testLanczos COMMAND testLanczos)
+
+
+# testArnoldiSaI executable
+add_compile_options(-Wall -Werror -g)
```

Comment:
CMake will add `-g` as appropriate for the build type.  But if you want to just apply this to all builds for this  test use
`target_compile_options(testArnoldSai PRIVATE -Wall -Werror -g)`